frontend/src/containers/home.tsx

```diff
@@ -71,6 +71,13 @@ const HomePage = () => {
     return themeContext.theme === LIGHT ? lightDeveloper : darkDeveloper;
   };
 
+  const [ourProjects, setOurProjects] = useState([]);
+  fetch("https://raw.githubusercontent.com/osscameroon/project-ideas/master/prod-projects.json")
```

Comment:
I am afraid this will cause some problem with github rate limits which are set to 60 requests per hour. 

We can probably not rely on this

Here is a discussion about that
https://github.community/t/raw-githubusercontent-com-rate-limit/142444/2